contrib/checkstyle/src/mill/contrib/checkstyle/CheckstyleModule.scala

```diff
@@ -0,0 +1,183 @@
+package mill
+package contrib.checkstyle
+
+import mill.api.{Loose, PathRef}
+import mill.scalalib.{DepSyntax, JavaModule, ScalaModule}
+import mill.util.Jvm
+
+import javax.xml.transform.TransformerFactory
+import javax.xml.transform.stream.{StreamResult, StreamSource}
+
+trait CheckstyleModule extends JavaModule {
```

Comment:
This `trait` and all the `def`s inside need some Scaladoc. Doesn't need to be anything fancy, just 1-2 lines summarizing what it is and what it's for would be enough to help future maintainers and users find their way around the codebase